source/mainnet/net/snippets/delegation-faq.rst

```diff
@@ -80,11 +80,11 @@ For delegators
 
 .. dropdown:: My account is suddenly delegating to passive delegation. Why is that?
 
-    It is likely because the baker pool to which you were delegating has been closed. You can continue to delegate to passive delegation or select a new baker pool for your delegation.
+    It is likely because the baker pool to which you were delegating has been closed. You can continue to delegate to :ref:`passive delegation<glossary-passive-delegation>` or select a new baker pool for your delegation.
```

Comment:
"likely" Isn't this the only reason? Or that the owner of the account has actively changed his/her delegation preferences.